include/iso15118/message/common_types.hpp

```diff
@@ -164,6 +165,11 @@ enum class GeneratorMode {
     GridForming = 2,
 };
 
+enum class GridCodeIslandingDetectionMethode {
```

Comment:
Nitpicking: `Method` instead of `Methode` (also in most variables with this type)
(Note that this was already a typo before this PR)

Reply:
Thx for noticing!